students.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] Great, you are making good progress and learning the concepts of `OOP` in the `ruby`. But I want to tell you that if you are passing the arguments `name`, `age` and `parent_permission` to the `super` method. It means it is coming from somewhere, otherwise, your application will break and give you an error. Kindly also take the age, name and `parent_permission` argument in your constructor of the class.